core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreReaderFragment.java

```diff
@@ -1038,6 +1039,7 @@ protected void closeFullScreen() {
     toolbarContainer.setVisibility(View.VISIBLE);
     updateBottomToolbarVisibility();
     exitFullscreenButton.setVisibility(View.GONE);
+    exitFullscreenButton.getBackground().setAlpha(255);
```

Comment:
Is this line fixing #2244 ? If not, I don't think it is necessary to change back to non-transparancy.

Reply:
nvm, read your commit message. 👍